properties/P000112.md

```diff
@@ -1,10 +1,12 @@
 ---
 uid: P000112
-aliases: []
 name: Submetrizable
+aliases:
+- Has a metrizable compression
+- Submetrisable
 refs:
-- mr: MR2492954
-  name: ! 'The compact-open topology: A new perspective'
-counterexamples_id: null
+- doi: 10.2307/2042236
```

Comment:
10.1090/S0002-9939-1978-0482639-9 seems a more accessible doi (directly to the AMS website)